src/js/components/NavButton.jsx

```diff
@@ -11,36 +11,38 @@ const NavButton = (props) => {
     macroVerified
   } = props
 
-  const disabled = ' usa-button-disabled'
-  let className = 'usa-button NavButton'
+  let className
   let suffix
-  let displayName
 
   switch (page) {
     case 'upload':
       suffix = 'syntacticalvalidity'
-      displayName = 'edits'
-      if(code < 8) className+=disabled
+      if(code < 8) className = 'usa-button-disabled'
       break
     case 'syntacticalvalidity':
       suffix = 'quality'
-      if(code < 8 || syntacticalValidityEditsExist) className+=disabled
+      if(code < 8 || syntacticalValidityEditsExist) className = 'usa-button-disabled'
       break
     case 'quality':
       suffix = 'macro'
-      if(code < 8 || syntacticalValidityEditsExist || !qualityVerified) className+=disabled
+      if(code < 8 || syntacticalValidityEditsExist || !qualityVerified) className = 'usa-button-disabled'
       break
     case 'macro':
       suffix = 'summary'
-      if(code < 8 || syntacticalValidityEditsExist || !qualityVerified || !macroVerified) className+=disabled
+      if(code < 8 || syntacticalValidityEditsExist || !qualityVerified || !macroVerified) className = 'usa-button-disabled'
       break
     default:
       return null
   }
 
-  if(!displayName) displayName = suffix
+  let displayName = (suffix === 'syntacticalvalidity') ? '' : suffix
+  displayName = (suffix !== 'summary') ? `${displayName} Edits` : displayName
```

Comment:
In this case it's just a `prop` passed to `RefileButton` and used [here](https://github.com/cfpb/hmda-platform-ui/pull/390/files#diff-06b62e091b18afbc21500ce204115596R4).